vault/resource_database_secret_backend_connection_test.go

```diff
@@ -723,6 +764,36 @@ resource "vault_database_secret_backend_connection" "test" {
 `, path, name, connURL, connLifetime, password)
 }
 
+func testAccDatabaseSecretBackendConnectionConfig_mysql_tls(name, path, connURL, password string, tls_ca string, tls_certificate_key string) string {
+	return fmt.Sprintf(`
+resource "vault_mount" "db" {
+  path = "%s"
+  type = "database"
+}
+
+resource "vault_database_secret_backend_connection" "test" {
+  backend = "${vault_mount.db.path}"
+  name = "%s"
+  allowed_roles = ["dev", "prod"]
+  root_rotation_statements = ["FOOBAR"]
+
+  mysql {
+	  connection_url = "%s"
+  }
+
+  data = {
+	  password            = "%s"
+	  tls_ca              = <<EOT
+%s
+EOT
+	  tls_certificate_key = <<EOT
+%s
+EOT
+  }
+}
+`, path, name, connURL, password, tls_ca, tls_certificate_key)
```

Comment:
And these need to be in the mysql block, which will change the parameter arg order a little:

```suggestion
  mysql {
	  connection_url = "%s"
	  tls_ca              = <<EOT
%s
EOT
	  tls_certificate_key = <<EOT
%s
EOT
  }

  data = {
	  password            = "%s"
  }
}
`, path, name, connURL, tls_ca, tls_certificate_key, password)
```